src/oasis/OASISLibrary_intern.ml

```diff
@@ -98,6 +98,17 @@ let schema, generator =
          s_ "Name used by findlib.")
       (fun (_, _, lib) -> lib.lib_findlib_name)
   in
+  let findlib_directory =
+    new_field schm "FindlibDirectory"
+      ~default:None
+      (* TODO: Check that the name is correct if this value is None, the
+               package name must be correct
+       *)
```

Comment:
Please add a "Feature:" to control this field. This is a new field and we need to test it at large before pushing it into a new version:
https://github.com/ocaml/oasis/blob/master/src/oasis/OASISFeatures.ml#L331